examples/durable/DurableApp/GetJobStatus/run.ps1

```diff
@@ -0,0 +1,8 @@
+param($name)
+
+if ((Get-Date).ToUniversalTime() -lt $name) {
```

Comment:
This looks a bit strange: you are comparing the current time with `$name`. Though technically legal, the intent is not clear. If you were going to encode the "desired" completion date in the name, you probably need to construct and pass this value in the MonitorStart code (I can see you are just passing `$null` there now), and then either give the parameter a better name or describe this in comments. Alternatively, consider returning "Not Completed" or "Completed" randomly, perhaps with 90/10% probability.